examples/remove-node-label.py

```diff
@@ -0,0 +1,19 @@
+from pprint import pprint
```

Comment:
We can add this to the previous example. right? in between we can also read the node and check to see if the label is there.